src/helpers/createTimeoutController.ts

```diff
@@ -1,6 +1,6 @@
-import { WaitOptions } from '../types'
+import { jestFakeTimersAreEnabled } from './jestFakeTimersAreEnabled'
 
-function createTimeoutController(timeout: WaitOptions['timeout']) {
+function createTimeoutController(timeout: number | boolean, allowFakeTimers: boolean) {
```

Comment:
`true` is not a valid value, only `number | false`

```suggestion
function createTimeoutController(timeout: number | false, allowFakeTimers: boolean) {
```

This would mean you would not need to cast it as a number below.

Reply:
👍